components/centraldashboard/app/server.ts

```diff
@@ -46,14 +47,20 @@ async function main() {
   app.use(express.json());
   app.use(express.static(frontEnd));
   app.use(attachUser(USERID_HEADER, USERID_PREFIX));
-  app.use(
-      '/api', new Api(k8sService, workgroupApi, metricsService).routes());
+  app.get('/debug', (req: Request, res: Response) => {
+    res.json({
+      user: req.user,
+      profilesServiceUrl,
+      codeEnvironment,
+    });
+  });
+  app.use('/api', new Api(k8sService, workgroupApi, metricsService).routes());
```

Comment:
I think this line is actually doing what I asked for above. In that case ignore my previous comment.